### DISCUSSION TITLE
KeePassXC Browser Extension

### DISCUSSION THREAD

**Discussion by mdoliver** [keepassxreboot/keepassxc#6208] [Ideas]
Hi all,

I'm using the Chrome KeePassXC Browser extension. Is anyone aware of plans to include a search function for your connected database directly from the browser extension?

I'm aware you can filter your password list in the extension but only after sites with valid URLs have been detected to complete credentials for.

For URLs that it cannot detect (eg a pop up login box) it would be great to search your dB from the extension and copy and paste usernames/passwords without having to leave your browser to search the KeePassXC app.

**Comment by droidmonkey** [comment]
We are not going to introduce that feature as it opens security issues. We limit retrieval of credentials to only those that match the current site you are on to limit database exposure within the browser.

**Reply by mdoliver** [reply]
Ok, thanks.
